### PR TITLE
Fix apc_* functions now return the value of functions they alias.

### PR DESCRIPTION
**Pull request by shouze**

### CODE REVIEW

File: polyfill.php [hunk 1]
```diff
@@ -3,76 +3,76 @@
...
     function apc_exists($keys)
     {
-        apcu_exists($keys);
+        return apcu_exists($keys);
     }
 }
 
 if (function_exists('apcu_fetch') && !function_exists('apc_fetch')) {
-    function apc_fetch($key, &$success)
+    function apc_fetch($key, &$success = null)
```
Comment (shouze) [inline polyfill.php:53]: Bonus one, was done for `apc_inc` but not for this function.

**Comment by lavoiesl** [comment]
Which version of PHP are you using? I’m not able to make the tests runs, the APC support seems to be always `Emulated`, so this polyfill doesn't do anything.

**Comment by shouze** [comment]
@lavoiesl I'm using php 7 with `git clone -b seven https://github.com/krakjoe/apcu`

BTW... this polyfill couldn't work as of course return of aliased functions were missing, you're agree with that no?

**Comment by lavoiesl** [comment]
Ok then. Thank you.

**Comment by shouze** [comment]
@lavoiesl in fact I've found how to have apc everytime, it's as simple as to pass following options when compiling apcu:

``` shell
$ ./configure --enable-apcu-bc --enable-apcu
```

**Comment by lavoiesl** [comment]
Yeah, my question was, how do you _not_ get apc-bc. I tried to reproduce it to be able to test that my polyfill is actually doing something, but APC support was always "Emulated".

**Comment by shouze** [comment]
@lavoiesl yes, I just compiled that `seven` branch of apcu with php 7 and with a bare configure (without any options passed).

**Pull request merged**